spec/factories/learning_hour_types.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :learning_hour_type do
+    casa_org { CasaOrg.first || create(:casa_org) }
+    name { Faker::Book.genre }
+    active { false }
```

Comment:
Should we just remove this and let it default to `active`, since that is the default in the DB?